docker-images/artifacts/kafka-thirdparty-libs/3.7.0/ignorelist

```diff
@@ -1,2 +1,2 @@
 jmx_prometheus_javaagent-1.0.1.jar
-snakeyaml-2.0.jar
\ No newline at end of file
+snakeyaml-2.2.jar
```

Comment:
is this update related to the config provider? Even if not, I am fine having it here.

Reply:
Yes. The config provider now doesn't depend on the snakeyaml 2.0 but 2.2 library. So I had to update the ignore list as well.